### PR TITLE
add gov proposal to lower deposit requirement

### PR DESCRIPTION
**Pull request by hxrts**
This proposal follows a discussion in the Governance Working Group Telegram about lowering the deposit requirements for moving a proposal into the voting period.

[rendered](https://github.com/cosmos/governance/blob/lower-deposit-req/proposals/2021-04-lower-deposit-requirement/README.md)

### CODE REVIEW

**Comment by zmanian** [comment]
I'm not sure if the subspace/key/value are correct.

Params confuses me still.

**Comment by hxrts** [comment]
I was going from [here](https://github.com/cosmos/governance/blob/master/params-change/Governance.md), but also asked @fedekunze to double check for me for this reason.

**Comment by hxrts** [comment]
Fixed (though someone please check)

File: proposals/2021-04-lower-deposit-requirement/proposal.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "title": "Parameter change: lower minimum proposal deposit amount",
+  "description": "The current deposit amount of 512 ATOMs prohibits valuable governance activity from small holders or those with most of their ATOM staked. We propose lowering the requirment to 64 ATOMS.",
+  "changes": [
+    {
+      "subspace": "mindeposit",
+      "key":"depositparams",
+      "value": "64000000"
```
Comment (fedekunze) [inline proposals/2021-04-lower-deposit-requirement/proposal.json:8]:
```suggestion
      "subspace": "gov",
      "key":"depositparams",
      "value": {
            "mindeposit": [
            {
                  "denom": "uatom",
                  "amount": "64000000"
            }
            ]
}
```

**Review by fedekunze — COMMENTED** [review]
I believe it should be something like this 👇. See https://github.com/cosmos/cosmos-sdk/issues/5800

File: proposals/2021-04-lower-deposit-requirement/proposal.json (hunk 1 above)
Comment (fedekunze) [reply proposals/2021-04-lower-deposit-requirement/proposal.json:8]: not sure if it resets the other deposit params tho...

**Comment by hxrts** [comment]
json from @ebuchman's [previous param change](https://github.com/cosmos/governance/blob/master/proposals/2020-10-blocks-per-year/proposal.json) reference. we really need better docs on this, yeesh.

**Comment by fedekunze** [comment]
> we really need better docs on this, yeesh.

It's only the governance params that are weird... Maybe we should also define a standard for keys (some modules use capitalized words and others all lower)

File: proposals/2021-04-lower-deposit-requirement/proposal.json (hunk 1 above)
Comment (fedekunze) [reply proposals/2021-04-lower-deposit-requirement/proposal.json:8]: Ok, so worst case the update will fail

Comment (hxrts) [reply proposals/2021-04-lower-deposit-requirement/proposal.json:8]: cool, this looks right based on the issue you linked. gov uses different format, bizarre. will maybe wait to get another thumbs up before committing.

Comment (alexanderbez) [reply proposals/2021-04-lower-deposit-requirement/proposal.json:8]: This is not fully correct. I wrote docs on this: https://github.com/cosmos/cosmos-sdk/blob/master/x/gov/spec/06_params.md

The key should be `min_deposit`, not `mindeposit`. JSON keys are snake-cased.

Also, per the doc above, you do not need to include the entire params JSON object, just the subset you want to change 👍

File: proposals/2021-04-lower-deposit-requirement/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+## Objectives
+
+1. Enable community members with good ideas but little capital to participate in governance and request resources from the community pool treasury.
+2. Improve the governance UX for holders who keep most of their ATOM staked.
+3. Increase utilization of treasury (currently 666,457 ATOM, approximately $14 MM USD, at time of writing).
+4. Accelerate Cosmos Hub development and growth.
+
+## Background
+
+Current deposit is 512 ATOMs (approximately $10k USD today). The ATOM price when the community treasury was activated (2019-05-03) was $4.99 (source: CoinMarketCap), meaning the total required deposit to submit a proposal was $2,555. Today, most proposers must coordinate with large ATOM holders to request additional funds in order to meet the minimum deposit requirements. This also applies to large ATOM holders who want to be active in governance but do not have enough liquid ATOM to meet the deposit requirements, as staked ATOM cannot be used to post deposits.
```
Comment (alexanderbez) [inline proposals/2021-04-lower-deposit-requirement/README.md:16]: While this is good motivation, we'll come across the same barrier to entry if the price of the ATOM token fluctuates greatly. What is really needed is an oracle to allows the minimum deposit to fluctuate and become elastic depending on the current price.

However, for the time being, this is sufficient.

Comment (hxrts) [reply proposals/2021-04-lower-deposit-requirement/README.md:16]: Totally agree, this is very much a temporary solution. I can indicate that in the proposal.

File: proposals/2021-04-lower-deposit-requirement/proposal.json (hunk 1 above)
Comment (hxrts) [reply proposals/2021-04-lower-deposit-requirement/proposal.json:8]: is `gov` the correct subspace?

Comment (alexanderbez) [reply proposals/2021-04-lower-deposit-requirement/proposal.json:8]: Yup!

Comment (hxrts) [reply proposals/2021-04-lower-deposit-requirement/proposal.json]: ok pushed this change

**Review by alexanderbez — APPROVED** [review]

**Review by ValarDragon — APPROVED** [review]

**Review by shahankhatch — APPROVED** [review]

**Comment by hxrts** [comment]
Wait @shahankhatch I thought we were only merging if a proposal had been submitted, or maybe even accepted? This one @fedekunze and I were going to submit today.

**Comment by shahankhatch** [comment]
In this instance, I suggest the addition of a status to the proposal so that its current state is recorded in the repo rather than relying on the PR/thread status to convey the proposal's status. This is used for many other draft specs/proposals/adrs.

**Pull request merged**